ui/src/components/Inventories/InventoryFormModal.js

```diff
@@ -0,0 +1,177 @@
+// import Box from '@material-ui/core'
+import Button from '@material-ui/core/Button'
+import Dialog from '@material-ui/core/Dialog'
+import DialogActions from '@material-ui/core/DialogActions'
+import DialogContent from '@material-ui/core/DialogContent'
+import DialogTitle from '@material-ui/core/DialogTitle'
+import Grid from '@material-ui/core/Grid'
+import React from 'react'
+import TextField from '../Form/TextField'
+import { Field, Form, Formik } from 'formik'
+import { MeasurementUnits } from '../../constants/units'
+import { Checkbox, FormControlLabel, MenuItem } from '@material-ui/core'
+
+const measurementUnitKeys = Object.keys(MeasurementUnits)
+const date = new Date()
+class InventoryFormModal extends React.Component {
+  render() {
+    const {
+      formName,
+      handleDialog,
+      products,
+      handleInventory,
+      title,
+      initialValues
+    } = this.props
+    return (
+      <Dialog
+        open={this.props.isDialogOpen}
+        maxWidth='sm'
+        fullWidth={true}
+        onClose={() => { handleDialog(false) }}
+      >
+        <Formik
+          initialValues={initialValues}
+          onSubmit={values => {
```

Comment:
In `onSubmit`, the `bestBeforeDate` is formatted as a `Date()` object to be viewed in the UI. This needs to be turned into an Instant/ISO String for the inventory to be created.

Change the `values.bestBeforeDate` into an ISOString with moment (source: https://momentjs.com/)

- Example of date conversion to ISO string: `moment("2022-11-30").toISOString()`